hw/ip/adc_ctrl/doc/_index.md

```diff
@@ -1,34 +1,215 @@
 ---
-title: "ADC_CTRL HWIP Technical Specification"
+title: "Analog to Digital Converter Control Interface"
 ---
 
 # Overview
 
-
+This document specifies the ADC controller IP functionality.
+This IP block implements control and filter logic for an analog block that implements a dual ADC.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
+See that document for integration overview within the broader top level system
 
 ## Features
 
+The IP block implements the following features:
 
+- Register interface to dual ADC analog block
+- Support for 2 ADC channels
+- Support for 8 filters on the values from the channels
+- Support ADCs with 10-bit output (two reserved bits in CSR)
+- Support for debounce timers on the filter output
+- Uses slow clock to enable use while the part is sleeping
+- Low power periodic scan mode for monitoring ADC channels
 
 ## Description
 
+The ADC controller is a simple front-end to an analog block that allows filtering and debouncing of the analog signals
+
+## Compatibility
 
+The ADC controller programming interface is not based on any existing interface
 
 # Theory of Operation
 
+The block diagram shows a conceptual view of the ADC controller state machine and filters
+
 ## Block Diagram
 
+![ADC_CTRL Block Diagram](adc_overview.svg "image_tooltip")
 
 ## Hardware Interface
 
 {{< incGenFromIpDesc "../data/adc_ctrl.hjson" "hwcfg" >}}
 
 ## Design Details
 
+## Sampling state machine
 
-# Programmers Guide
+The state machine that takes ADC samples follows a very simple pattern:
+
+1. *Power up ADC*: The controller issues the power up command to the ADC.
+
+2. *Wait for ADC turn on*: The controller waits for the number of clock cycles programmed in {{< regref "adc_pd_ctl.pwrup_time" >}} which should be set to match the ADC power up delay.
+
+3. *Take sample Channel 0*: The ADC is requested to sample channel 0. When the ADC signals complete the value is stored in {{< regref "adc_chn_val[0].adc_chn_value" >}}. Note that the time taken in this step depends on the properties of the ADC.
```

Comment:
Nit: put each sentence on a new line, e.g. as you have done for point 6. below (this should still group the text below a bullet point).

Reply:
Sure.